js/components/directionControlVr.jsx

```diff
@@ -19,7 +20,15 @@ class DirectionControlVr extends React.Component {
    * @param {Event} event - event object
    */
   rotateVrVideo(event) {
-    const isRotated = event && (event.type === 'mousedown' || event.type === 'touchstart');
+    const charCode = event.which || event.keyCode;
+    const enterCharCode = 13;
+    const spaceCharCode = 32;
```

Comment:
Keycodes constants are here:
https://github.com/ooyala/html5-skin/blob/master/js/constants/constants.js#L252-L261

Reply:
@zelibobla oh, yes, exactly. Thank you, Anton!